classes/hypeJunction/Scraper/Services/Linkify.php

```diff
@@ -165,7 +165,7 @@ public function callbackUsername($matches) {
 			'data-value' => $username,
 			'data-guid' => $user->guid,
 			//'data-icon' => $user->getIconURL('tiny')
-				), $user->getDisplayName());
+				), $user->name);
```

Comment:
You could do that but does not add much, since this patch is for the 1.8 branch, where it is for sure not callable.